articles/how-to-guides/create-qsharp-project.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Learn how to create a Q# project with the Quantum Development Kit (QDK)
+description: Initialize a project for quantum development with the QDK and Q# in the development environment of you choice
+author: natke
+ms.author: nakersha
+ms.date: 10/19/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.howto.createproject
+---
+
+# Create a Q# project in your development environment
+
+Learn how to create a Q# project with the QDK.
+
+A Q# project contains Q# files containing quantum code, as well as a host program to run the quantum program. You can write the host program in .NET languages such as C#, or in Python. You can also run Q# code in a Jupyter notebook using the IQ# kernel.
+
+Choose your development environment and language from the sections below.
```

Comment:
Would it be possible to add a bullet list of anchor links here?  I think it'd really add to the quick-accessibility, because the eye really doesn't go naturally to the automatically generated ones in the right-side margin (under "In this article").

Reply:
That's a good idea, yes